src/attach.rs

```diff
@@ -40,6 +40,8 @@ pub fn attach(opts: &AttachOptions) -> Result<()> {
         run_kvm_wrapped(&vm, &device),
         "device init stage with KvmRunWrapper failed"
     );
+    println!("blkdev queue ready.");
```

Comment:
Eventually it would be good to put all these `println!()` into log functions so we can disable them by default. Maybe log crate with a custom log handler.

Reply:
:) true. i just added great log infrastructure in the last commit